tasks/testwasher.py

```diff
@@ -92,14 +90,28 @@ def process_module_results(self, module_results: list[ModuleTestResult]):
 
         should_succeed = True
         failed_tests = []
+        failed_command_modules = []
         for module_result in module_results:
-            if non_flaky_failing_tests := self.get_non_flaky_failing_tests(module_result.path):
+            failing_tests, flaky_marked_tests = self.parse_test_results(module_result.path)
+            non_flaky_failing_tests = self.get_non_flaky_failing_tests(
+                failing_tests=failing_tests, flaky_marked_tests=flaky_marked_tests
+            )
+            if (
+                not failing_tests and module_result.failed
+            ):  # In this case the Go test command failed on one of the modules but no test failed, it means that the test command itself failed (build errors,...)
+                should_succeed = False
+                failed_command_modules.append(module_result.path)
+            if non_flaky_failing_tests:
                 should_succeed = False
                 for package, tests in non_flaky_failing_tests.items():
                     failed_tests.extend(f"- {package} {test}" for test in tests)
         if failed_tests:
             print("The test command failed, the following tests failed and are not supposed to be flaky:")
             print("\n".join(sorted(failed_tests)))
+        if failed_command_modules:
+            print("The test command failed, before test execution on the following modules:")
+            print("\n".join(sorted(failed_command_modules)))
+            print("Please check the job logs for more information")
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:**  Never caught this before but it might be nice to put this in `stderr` until we use a proper logger.